phpBB/includes/acp/acp_styles.php

```diff
@@ -756,22 +756,26 @@ protected function find_available($all)
 				// Style is already installed
 				continue;
 			}
-			$cfg = $this->read_style_cfg($dir);
-			if ($cfg === false)
+
+			try
+			{
+				$style_data = $this->read_style_composer_file($dir);
+			}
+			catch (\DomainException $e)
 			{
-				// Invalid style.cfg
+				// Invalid composer.json
```

Comment:
why do we skip the exception? if the composer.json is invalid the following checks may error